Creestoph-solutions/Classroom.py

```diff
@@ -0,0 +1,42 @@
+class AlreadyInClassException(Exception):
+    pass
+
+
+class Classroom(dict):

```

Comment:
Bardzo fajny pomysł, tutaj można się pokusić o nadpisanie operatorów słownika, zamiast pisania nowych funkcji `add_student` itd.

http://stackoverflow.com/questions/2328235/pythonextend-the-dict-class